pkgs/development/python-modules/graphqlclient/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "graphqlclient";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  checkInputs = [
+      six
+    ];
+
+  meta = with lib; {
+    description = "Simple GraphQL client for Python 2.7+";
```

I don't think that version details add value in the description.

```suggestion
    description = "Simple GraphQL client for Python";
```